services/docker-compose.yml

```diff
@@ -297,6 +298,25 @@ services:
     networks:
       - default
       - interactive_services_subnet
+  wb-db-event-listener:
+    image: ${DOCKER_REGISTRY:-itisfoundation}/webserver:${DOCKER_IMAGE_TAG:-latest}
+    init: true
+    hostname: "{{.Node.Hostname}}-{{.Service.Name}}-{{.Task.Slot}}"
+    environment:
+      <<: *webserver-environment
+      WEBSERVER_DB_LISTENER: 1
+    env_file:
+      - ../.env
+      - ../.env-wb-db-event-listener
+    deploy:
+      # NOTE: this MUSTN'T change, or weird things might happen
+      # this will stay until all legacy dynamic services are gone.
+      replicas: 1
```

Comment:
So even after using the `AsyncAioPikaManager` we can't still scale it up?
This seems counter intuitive.